### ISSUE TITLE
Can't bind to 'ngIf' since it isn't a known property of 'span'

### ISSUE THREAD

**Issue by bentzibentz** [videogular/ngx-videogular#99]
### Description
Since angular > v14 we get the following error messages:

`Can't bind to 'ngIf' since it isn't a known property of 'span' (used in the 'VgScrubBarCurrentTimeComponent' component template). If the 'ngIf' is an Angular control flow directive, please make sure that the 'CommonModule' is a part of an @NgModule where this component is declared.` 

`Can't bind to 'ngIf' since it isn't a known property of 'span' (used in the 'VgTimeDisplayComponent' component template).
If the 'ngIf' is an Angular control flow directive, please make sure that the 'CommonModule' is a part of an @NgModule where this component is declared.`

Scrub Bar and Time Display are not working anymore.

The integration is done as shown in the documentation: https://videogular.github.io/ngx-videogular/docs/modules/controls/vg-scrub-bar/

Any ideas?

**Comment by nzomedia** [comment]
Hello everyone,
i'm having a similar issue:

```
NG0303: Can't bind to 'ngIf' since it isn't a known property of 'span' (used in the 'VgScrubBarCurrentTimeComponent' component template).
If the 'ngIf' is an Angular control flow directive, please make sure that the 'CommonModule' is a part of an @NgModule where this component is declared.
```

In the browser console, i can see the issue comes from this line:
```VgScrubBarCurrentTimeComponent_Template videogular-ngx-videogular-controls.mjs:1743```

I'm using Angular@14, ngx-videogular@5.0.1.

I can play some mp4, but the error messages show when the player starts playing.

**Comment by hochstibe** [comment]
I had the same issue with Angular 14 + ngx-videogular 5.0.1, updating to 6.0.0 resolved the issue.

**Comment by nzomedia** [comment]
> I had the same issue with Angular 14 + ngx-videogular 5.0.1, updating to 6.0.0 resolved the issue.

Thank you, it works without errors now.

**Comment by bentzibentz** [comment]
Seems to be fixed in v6.0.0
